packages/spotlight/SpotlightContainerDecorator/SpotlightContainerDecorator.js

```diff
@@ -190,6 +210,9 @@ const SpotlightContainerDecorator = hoc(defaultConfig, (config, Wrapped) => {
 
 		/**
 		 * Whether or not the container is in muted mode.
+		 * When in muted mode, Spottable controls within the container can still gain focus,
+		 * however their `:focus` CSS styles will not be applied, giving them the appearance of not
```

Comment:
can still gain focus, however their
-> can still gain focus but, their

Reply:
Hm, You mean "can still gain focus, but their"??
I fixed to "can still gain focus but, their"